rpcs3/rpcs3qt/log_frame.cpp

```diff
@@ -467,23 +479,59 @@ void log_frame::UpdateUI()
 			// clear selection or else it will get colorized as well
 			text_cursor.clearSelection();
 
-			// write text to the end
-			text_cursor.movePosition(QTextCursor::End);
-			text_cursor.insertText(qstr(buf));
-
-			// if we mark text from right to left we need to swap sides (start is always smaller than end)
-			if (sel_pos < sel_end)
+			std::stringstream buf_stream;
+			buf_stream.str(buf);
+			std::string buf_line;
+			while(std::getline(buf_stream, buf_line)) 
 			{
-				std::swap(sel_start, sel_end);
-			}
+				QString suffix;
+				QString tty_text = QString::fromStdString(buf_line);
+				std::string debugstring = "no selection";
```

Comment:
Idk what this is

Reply:
my inability to proofread. Must have forgotten to delete it after debugging.